api-report/tree.api.md

```diff
@@ -226,11 +221,11 @@ export interface IEditableForest extends IForestSubscription {
 // @public
 export interface IForestSubscription extends Dependee {
     allocateCursor(): ITreeSubscriptionCursor;
-    root(range: DetachedField): ForestAnchor;
+    root(range: DetachedField): Anchor;
     // (undocumented)
     readonly rootField: DetachedField;
     readonly schema: SchemaRepository & Dependee;
-    tryGet(destination: ForestAnchor, cursorToMove: ITreeSubscriptionCursor, observer?: ObservingDependent): TreeNavigationResult;
+    tryMoveCursorTo(destination: Anchor, cursorToMove: ITreeSubscriptionCursor, observer?: ObservingDependent): TreeNavigationResult;
```

Comment:
This doesn't need to be addressed in this PR, but why can't the `tryMoveCursor` operation be exposed on the cursor to be moved?

Reply:
which things are on forest vs the cursor are pretty arbitrary. We should probably revisit that when we try and do documentation for the observation tracking, and see if we can structure things in a way thats more clear.